packages/cactus-cmd-api-server/src/main/typescript/api-server.ts

```diff
@@ -223,14 +228,6 @@ export class ApiServer {
         this.log.info(`Cactus API reachable ${httpUrl}`);
       }
 
-      {
-        const { cockpitHost: host } = this.options.config;
-        const { port } = addressInfoCockpit;
-        const protocol = cockpitTlsEnabled ? "https:" : "http:";
-        const httpUrl = `${protocol}//${host}:${port}`;
-        this.log.info(`Cactus Cockpit reachable ${httpUrl}`);
```

Comment:
@AzaharaC Could we still log this based on the `cockpitEnabled` flag being set to true?